core/editor/editor-v2-packages.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Elementor\Core\Editor;
+
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Core\Utils\Collection;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Editor_V2_Packages {
```

Comment:
Delete the class and move the filter to the enueue packages method of the v2_loader and make it static.

Reply:
Done.